app/editor/src/features/content/morning-report/AdvancedMorningReport.tsx

```diff
@@ -0,0 +1,93 @@
+import { ToolBarSection } from 'components/tool-bar';
+import { fieldTypes } from 'features/content/list-view/constants';
+import { IContentListAdvancedFilter } from 'features/content/list-view/interfaces';
+import React from 'react';
+import { FaArrowAltCircleRight, FaBinoculars } from 'react-icons/fa';
+import { useContent } from 'store/hooks';
+import {
+  FieldSize,
+  fromQueryString,
+  instanceOfIOption,
+  IOptionItem,
+  OptionItem,
+  replaceQueryParams,
+  Row,
+  Select,
+  Text,
+} from 'tno-core';
+
+import { IMorningReportFilter } from './interfaces';
+
+export interface IAdvancedSearchSectionProps {
+  onSearch: (filter: IMorningReportFilter & IContentListAdvancedFilter) => void;
+}
+
+/**
+ *
+ * @param onSearch Inform the parent of the request to search.
+ * @returns Filter section containing the advanced filter
+ */
+export const AdvancedMorningReport: React.FC<IAdvancedSearchSectionProps> = ({ onSearch }) => {
+  const [{ morningReportFilter, filterAdvanced, filter }, { storeFilterAdvanced }] = useContent();
+  const search = fromQueryString(window.location.search);
+  /** initialize advanced search section with query values or new */
+  React.useEffect(() => {
+    storeFilterAdvanced({
+      ...filterAdvanced,
+      fieldType: search.fieldType ?? fieldTypes[0].value,
+      searchTerm: search.searchTerm ?? '',
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Ignoring dependencies is almost always wrong.  Is this extracting the URL parameters the first time it loads only?

Reply:
Yes this was to extract everything from the URL and match it to the filter when the user first goes to the page (if they have a pre existing link for example)